Gemfile

```diff
@@ -0,0 +1,3 @@
+source "https://rubygems.org"                                                                                                                                                                                                                                                          
+
+gem 'cocoapods', '0.39.0'
```

Comment:
Another annoying request to use double-quotes here. I'm not really picky about which quotes are used in ruby, just that they are consistently used. I have a problem, I know.